src/components/Correction/CompletingParty.vue

```diff
@@ -57,12 +55,8 @@ export default class CompletingParty extends Mixins(CommonMixin) {
   get originalCompletingParty () : OrgPersonIF {
     if (this.getOriginalIA) {
       return this.getCompletingParty(this.getOriginalIA.parties)
-    } else if (this.getOriginalChgRegistration) {
-      return this.getCompletingParty(this.getOriginalChgRegistration.parties)
-    } else if (this.getOriginalRegistration) {
-      return this.getCompletingParty(this.getOriginalRegistration.parties)
     } else {
-      return null
+      return this.getCompletingParty(this.getEntitySnapshot?.orgPersons)
```

Comment:
This component is actually not used for firm corrections so the old code was superfluous.

Still, this seems like a reasonable fallback for any future (non BEN IA) corrections that might use this component.